scripts/make.sh

```diff
@@ -32,7 +32,8 @@ for var in "$@"; do
             catkin_args=( "${catkin_args[@]}" "-p$(distcc -j) -j$(distcc -j) --no-jobserver" )
             ;;
         *)
-            catkin_args=( "${catkin_args[@]}" "$var" )
+            dirs=$(find $ROSWSS_ROOT/src -path "**/$var/package.xml" -exec bash -c 'basename "$(dirname {})"' \;)
+            catkin_args=( "${catkin_args[@]}" ${dirs} )
```

Comment:
Thanks for your contribution! The "otherwise" branch is supposed to forward regular [catkin build arguments](https://catkin-tools.readthedocs.io/en/latest/verbs/catkin_build.html#full-command-line-interface). As far I understand, your proposed code would break this behavior. I'm afraid it needs more work to distuingish your use case and catkin argument forwarding.

Reply:
Thank you for pointing out the issue with my proposed changes disrupting the forwarding of regular catkin build arguments. I appreciate your feedback. I've taken it into consideration and have made a new commit that I believe will address this concern.
In the updated implementation, even if a ROS package is not found for a given argument, that argument will still be included. This way, it ensures the regular catkin arguments are not disrupted and continues to be forwarded as expected.